frontend/src/components/MeetingRoom/EmojiGrid/EmojiGrid.tsx

```diff
@@ -29,13 +29,11 @@ const EmojiGrid = ({
 }: EmojiGridProps): ReactElement => {
   const isSmallViewport = useIsSmallViewport();
 
-  const handleClickAway = (event: MouseEvent | TouchEvent) => {
-    const target = event.target as HTMLElement;
-
-    if (isSmallViewport && !target.closest('#emoji-grid-toggle')) {
```

Comment:
Missed during code review, but found during `Cmd + F` - we don't actually have an id of `emoji-grid-toggle` 😂